internal/plugins/info.csv

```diff
@@ -88,6 +88,7 @@ gcp_cloud_storage         ,output    ,GCP Cloud Storage         ,3.43.0  ,certif
 gcp_cloudtrace            ,tracer    ,GCP Cloud Trace           ,4.2.0   ,certified  ,n          ,y     ,y
 gcp_pubsub                ,input     ,GCP PubSub                ,0.0.0   ,certified  ,n          ,y     ,y
 gcp_pubsub                ,output    ,GCP PubSub                ,0.0.0   ,certified  ,n          ,y     ,y
+gcp_spanner               ,input     ,gcp_spanner               ,0.0.0   ,community  ,n          ,n     ,n
```

Comment:
- We should update the version here (can wait but needs to be right before merged)
- This is enterprise not community
- It should be available in cloud (both last two columns ticked yes

Reply:
Done